docs/integrations/microsoft-azure/opentelemetry/sql-server-linux-opentelemetry.md

```diff
@@ -0,0 +1,138 @@
+description: Learn about the Sumo Logic OpenTelemetry app for Microsoft SQL Server for Linux.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/sql.png')} alt="thumbnail icon" width="50"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The Sumo Logic app for Microsoft SQL Server is a logs based app that provides insight into your SQL Server for Linux. The app consists of predefined dashboards, providing visibility into your environment for real-time or historical analysis on backup, restore mirroring, general health and operations of your system.
+
+This app has been tested with following SQL Server versions:
+
+- Microsoft SQL Server 2016
+
+SQL Server logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/SQLServer-Linux-OpenTelemetry/SQL-Server-Schematics.png' alt="Redis Logs dashboards" />
+
+## Fields creation in Sumo Logic for SQL Server
+
+Following are the [Fields](https://help.sumologic.com/docs/manage/fields/) which will be created as part of SQL Server app install if not already present.
+
+* `db.cluster.name`. User configured. Enter a name to identify this SQL Server cluster. This cluster name will be shown in the Sumo Logic dashboards.
+* `db.system`. Has a fixed value of **sqlserver**.
+* `deployment.environment`. User configured. This is the deployment environment where the SQL Server cluster resides. For example dev, prod, or qa.
+* `sumo.datasource`. Has a fixed value of **sqlserver**.
+
+## Prerequisites
+
+Make sure logging is turned on in SQL Server. Follow [this documentation](https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/scm-services-configure-sql-server-error-logs?view=sql-server-ver15) to enable it.
+
+The Microsoft SQL Server app's queries and dashboards depend on logs from the SQL Server ERRORLOG, which is typically found in: `/var/opt/mssql/log/errorlog`.
+
+The ERRORLOG is typically in UTF-16LE encoding, but verify the file encoding used in your SQL Server configuration.
+
+## Configure SQL Server Logs Collection
+
+### Step 1: Set up Collector
+
+If you want to use an existing Otel Collector then this step can be skipped by selecting the option of using an existing Collector.
+
+If you want to create a new Collector, select the **Add a new Collector** option.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/SQLServer-OpenTelemetry/SQL-Server-Collector.png' alt="Collector" />
```

Comment:
```suggestion
1. If you want to create a new Collector, select the **Add a new Collector** option.
1. Select the platform for which you want to install the Sumo OpenTelemetry Collector. <br/>This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent. <br/><img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/SQLServer-OpenTelemetry/SQL-Server-Collector.png' alt="Collector" />
```

Reply:
This should be a numbered list.